### PR TITLE
Add new models, datasets, videos to task pages

### PR DESCRIPTION
**Pull request by merveenoyan**
I updated many models, added datasets and resources.
I also added two videos that I shot on text-generation and image-text-to-text generation with Hugging Face for their associated task pages. Although multimodal one is not only it2t it's mostly focused on tooling around it. 

In case you want to take a look:
https://www.youtube.com/watch?v=e9gNEAlsOvU
https://www.youtube.com/watch?v=IoGaGfU1CIg

### CODE REVIEW

File: packages/tasks/src/tasks/text-generation/data.ts [hunk 1]
```diff
@@ -51,32 +55,32 @@ const taskData: TaskDataCustom = {
...
 		},
 		{
 			description: "A large code generation model that can generate code in 80+ languages.",
 			id: "bigcode/starcoder",
 		},
 		{
-			description: "A very powerful text generation model.",
-			id: "mistralai/Mixtral-8x7B-Instruct-v0.1",
+			description: "A very powerful text generation model trained to follow instructions.",
+			id: "meta-llama/Meta-Llama-3.1-8B-Instruct",
```
Comment (Vaibhavs10) [inline packages/tasks/src/tasks/text-generation/data.ts:67]:
```suggestion
			id: "meta-llama/Meta-Llama-3.1-405B-Instruct",
```

For the LOLs, it technically is the most powerful open text generation model hehe.

**Review by Vaibhavs10 — APPROVED** [review]
Thanks a ton for updating all the tasks pages! ❤️

File: packages/tasks/src/tasks/text-to-image/data.ts [hunk 2]
```diff
@@ -46,15 +46,15 @@ const taskData: TaskDataCustom = {
...
 		},
 		{
 			description: "A powerful yet fast image generation model.",
 			id: "latent-consistency/lcm-lora-sdxl",
 		},
 		{
-			description: "A very fast text-to-image model.",
-			id: "ByteDance/SDXL-Lightning",
+			description: "Text-to-image model for photorealistic generation.",
+			id: "Kwai-Kolors/Kolors",
```
Comment (pcuenca) [inline packages/tasks/src/tasks/text-to-image/data.ts:57]: Not super heavily used yet, is it?

File: packages/tasks/src/tasks/text-to-image/data.ts [hunk 3]
```diff
@@ -46,15 +46,15 @@ const taskData: TaskDataCustom = {
 	models: [
 		{
 			description: "One of the most powerful image generation models that can generate realistic outputs.",
-			id: "stabilityai/stable-diffusion-xl-base-1.0",
+			id: "black-forest-labs/FLUX.1-dev",
```
Comment (pcuenca) [inline packages/tasks/src/tasks/text-to-image/data.ts:49]: 👍

**Review by pcuenca — COMMENTED** [review]
Nice, not familiar with all the selections, but it's super cool to keep improving these pages.

File: packages/tasks/src/tasks/text-generation/data.ts [hunk 4]
```diff
@@ -51,32 +55,32 @@ const taskData: TaskDataCustom = {
 	],
 	models: [
 		{
-			description: "A large language model trained for text generation.",
-			id: "bigscience/bloom-560m",
+			description: "A large text-generation model trained to follow instructions.",
```
[the remark this answers is not in the document]
Comment (merveenoyan) [reply packages/tasks/src/tasks/text-generation/data.ts]: addressed

File: packages/tasks/src/tasks/text-to-image/data.ts (hunk 2 above)
Comment (merveenoyan) [reply packages/tasks/src/tasks/text-to-image/data.ts:57]: I saw @apolinario was promoting it a month ago or two so I thought I'd add

File: packages/tasks/src/tasks/text-generation/data.ts (hunk 1 above)
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/text-generation/data.ts:67]: yeah makes sense!

**Review by pcuenca — APPROVED** [review]

**Pull request merged**